### PR TITLE
Review ramlconv and make improvements to package

### PR DESCRIPTION
**Pull request by edwardjrp**
- Update setup.py
- Fix pip install issues
- Add non python code assets as manifest.in resources
- Fix some workflow issues

### CODE REVIEW

File: ramlpreparer/config.py [hunk 1]
```diff
@@ -59,8 +59,10 @@ def apply_file(self, f):
         recognized settings found there to this configuration.
         Environment variables take precedence over any values found here.
         '''
-        with open(f, 'r') as deconst_file:
+
+        with open(DECONST_FILE, "r", encoding="utf-8") as deconst_file:
```
Comment (nimbinatus) [inline ramlpreparer/config.py]: Change this one back from `DECONST_FILE` to `f` or change the variable call on line 56 in the method definition. This switch is causing the tests to fail to find the `_deconst.json` file.

File: ramlpreparer/deconstraml.py [hunk 2]
```diff
@@ -52,7 +50,7 @@ def find_all(config):
     '''
     listed_raml = []
     excluded_filepath = set(['node_modules'])
-    for (dirpath, dirnames, filenames) in os.walk(config.git_root, topdown=True):
+    for (dirpath, dirnames, filenames) in os.walk(config.content_root, topdown=True):
```
Comment (nimbinatus) [inline ramlpreparer/deconstraml.py:53]: Any reason for switching this?

**Review by nimbinatus — DISMISSED** [review]
Adjustments due to failed unit tests. Otherwise, the only other failed test seems to be an issue with how https://github.com/nimbinatus/deconst-raml-preparer/blob/master/tests/test_envelope_writer.py#L51 finds the files (it's finding `asset_test_html.html` instead of `test_html.html`). Doesn't seem to be related to your updates, but maybe I'm missing something?

File: ramlpreparer/deconstraml.py (hunk 2 above)
Comment (edwardjrp) [reply ramlpreparer/deconstraml.py:53]: If i recall well there was another value going in git root, and wasnt making sense.

File: ramlpreparer/config.py (hunk 1 above)
Comment (edwardjrp) [reply ramlpreparer/config.py]: Changed to f, the value previously was just the string '_deconst.json'

**Comment by nimbinatus** [comment]
Tests should get fixed when #2 lands (at least, they do locally...)

**Review by nimbinatus — APPROVED** [review]
+1, LGTM! Thanks for fixing the test :)

**Pull request merged**